src/gamekit.cs

```diff
@@ -791,11 +794,20 @@ interface GKLocalPlayer
 		[Export ("unregisterAllListeners")]
 		void UnregisterAllListeners ();
 
+		[Deprecated (PlatformName.iOS, 13, 4, message: "Use 'FetchItemsForIdentityVerificationSignature' along with 'GKPlayer.TeamPlayerId' instead.")]
+		[Deprecated (PlatformName.TvOS, 13, 4, message: "Use 'FetchItemsForIdentityVerificationSignature' along with 'GKPlayer.TeamPlayerId' instead.")]
+		[Deprecated (PlatformName.MacOSX, 10, 15, 4, message: "Use 'FetchItemsForIdentityVerificationSignature' along with 'GKPlayer.TeamPlayerId' instead.")]
+		[Deprecated (PlatformName.WatchOS, 6, 2, message: "Use 'FetchItemsForIdentityVerificationSignature' along with 'GKPlayer.TeamPlayerId' instead.")]
```

Comment:
why the `along with 'GKPlayer.TeamPlayerId'` ?
that's not something used as an argument
how does that differ from the old API ?

Reply:
Looks like I am not at paraphrasing 

```
API_DEPRECATED("API deprecated. Use fetchItemsForIdentityVerificationSignature: and the teamPlayerID value to verify a user identity.", ios(7.0, 13.5), tvos(9.0, 13.5), macosx(10.10, 10.15.5), watchos(3.0,6.5));
```

This is because in code you use the callback params (publicKeyUrl, signature, salt, and timestamp) along with TeamPlayerId to create a payload and verify the user against a web service.

I guess that goes beyond the deprecation API message and can remove if you think is best.

Reply:
I'd rather stick with our default `Use 'X' instead.` unless the information is complete enough to avoid a trip to the API documentation (ours or Apple).

Reply:
@spouliot fixed